### DISCUSSION TITLE
Nightly builds of Infinity for F-droid

### DISCUSSION THREAD

**Discussion by tabasavr** [Docile-Alligator/Infinity-For-Reddit#1302] [Ideas]
It is possible to set up automated builds of development branch and publish them as nightly version: 
https://f-droid.org/docs/Publishing_Nightly_Builds

This would help test new features and find/fix bugs quicker.

**Comment by Wladefant** [comment]
Yes, I would very much support that and will add the file, if that fits into the plans of @Docile-Alligator But at the moment we don't have a development branch. Would you like one?

**Reply by tabasavr** [reply]
> But at the moment we don't have a development branch.

I meant master

**Reply by Wladefant** [reply]
Yeah I know, but would you like one?

**Reply by tabasavr** [reply]
Discussed in DMs, a separate dev branch is not needed unless it solves some problem.
